lib/orderbook/OrderBook.ts

```diff
@@ -286,7 +295,13 @@ class OrderBook extends EventEmitter {
       this.localIdMap.set(localId, order.id);
     }
 
-    this.getOrderMap(ordersMap, order).set(order.id, order);
+    const orderMap = this.getOrderMap(ordersMap, order);
+    if (orderMap.has(order.id)) {
+      return false;
+    } else {
+      orderMap.set(order.id, order);
+      return true;
```

Comment:
Could you add a comment block to this method explaining that it attempts to add an order if none exists in the ordermap, `@returns true if the order was added, false if the order is a duplicate`.

Reply:
In this case it's specifically a duplicated order id, so I changed your text a bit